guacamole-docker/bin/start.sh

```diff
@@ -363,24 +363,47 @@ END
         exit 1;
     fi
 
+    # Handle Docker secrets for LDAP
+    if [ -n "$LDAP_SEARCH_BIND_DN_FILE" ]; then
+        set_optional_property "ldap-search-bind-dn" "$(cat $LDAP_SEARCH_BIND_DN_FILE)"
+    elif [ -n "$LDAP_SEARCH_BIND_DN" ]; then
+        set_optional_property "ldap-search-bind-dn" "$LDAP_SEARCH_BIND_DN"
+    fi
+
+    if [ -n "$LDAP_SEARCH_BIND_PASSWORD_FILE" ]; then
+        set_optional_property "ldap-search-bind-password" "$(cat $LDAP_SEARCH_BIND_PASSWORD_FILE)"
+    elif [ -n "$LDAP_SEARCH_BIND_PASSWORD" ]; then
+        set_optional_property "ldap-search-bind-password" "$LDAP_SEARCH_BIND_PASSWORD"
+    fi
+
+    if [ -n "$LDAP_USER_BASE_DN_FILE" ]; then
```

Comment:
Is there a particular reason to include the various DNs in the Docker secrets support? I understand for password, and possibly for username, but it doesn't seem like an LDAP DN is all that sensitive a value that it needs to be protected??